packages/utils-telemetry/src/telemetry-api.ts

```diff
@@ -3,30 +3,23 @@ type Properties = { [key: string]: string };
 
 type TelemetryItem = {
     data: {
-        baseData: {
-            measurements: Measurements;
-            name: string;
-            properties: Properties;
-            ver?: number;
-        };
-        baseType: string;
+        measurements: Measurements;
+        name: string;
+        properties: Properties;
     };
-    iKey: string;
-    name: string;
+    type: string;
     time: string;
 };
 
-const appInsightsEndpoint = 'https://dc.services.visualstudio.com/v2/track';
+const telemetryEndPoint = 'https://webhint-telemetry.azurewebsites.net/api/log';
 
-let nameKey = '';
 let sendTimeout: NodeJS.Timeout | null = null;
 let telemetryQueue: TelemetryItem[] = [];
 
 let options = {
     batchDelay: 15000,
     defaultProperties: {} as Properties,
     enabled: false,
-    instrumentationKey: '8ef2b55b-2ce9-4c33-a09a-2c3ef605c97d',
```

Comment:
Should we do any type of validation to make sure we are not getting spammed?

Reply:
I also had this discussion with Tony, and the gist of it was that even AI endpoint was public and we could have been spammed before. So, we thought we wouldn't need it, but if we do feel like we want to track some spamming, we should do it as part a different task.

Reply:
we had an ID but agree that doesn't seem worth it :)

Reply:
We will notice a deliberate spamming attempt in the chart for sure, but it will be more of a reactive feature then, rather than a proactive feature if we do it right now. We can definitely keep an eye on it to see if it is necessary.